linode/manager
# [OBJ] Refresh settings on creation success and failure

## Description

**[USE DEV OR STAGING]**

It turns out that when creating a bucket for the first time, Object Storage is enabled even if the bucket creation fails (same for Access Keys).

To test this:

1. Have OBJ disabled.
2. Create a bucket named “test”.
3. Click “Enable Object Storage” in the confirmation modal.
4. The Bucket creation fails since the name is not unique.
5. Curl account/settings. 
6. Observe: object_storage is ‘active’.

In this PR, we request accountSettings on bucket/access key creation success or failure, if Object Storage was previously disabled.

Note: I also made a copy change to the Cancellation flow, which Marty suggested in https://github.com/linode/manager/pull/5653.
